main/acle.md

```diff
@@ -9382,6 +9416,2341 @@ are named after. All of the functions have external linkage.
+  __attribute__((arm_streaming))
+  svfloat16x2_t svmin[_f16][_x2](svfloat16x2_t zdn, svfloat16x2_t zm);
+  
+  
+  // Variants are also available for _f32, _f64, _s8, _s16, _s32, _s64, _u8, _u16,
+  // _u32 and _u64
+  __attribute__((arm_streaming))
+  svfloat16x4_t svmin[_f16][_x4](svfloat16x4_t zdn, svfloat16x4_t zm);
+  
+  ```
+
+#### FMAXNM, FMINNM (single)
+
+Multi-vector floating point min/max number
+
+``` c
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x2_t svmaxnm[_single][_f16][_x2](svfloat16x2_t zdn, svfloat16_t zm);
+  
+  
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x4_t svmaxnm[_single][_f16][_x4](svfloat16x4_t zdn, svfloat16_t zm);
+  
+  
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x2_t svminnm[_single][_f16][_x2](svfloat16x2_t zdn, svfloat16_t zm);
+  
+  
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x4_t svminnm[_single][_f16][_x4](svfloat16x4_t zdn, svfloat16_t zm);
+  
+  ```
+
+#### FMAXNM, FMINNM (multi)
+
+Multi-vector floating point min/max number
+
+``` c
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x2_t svmaxnm[_f16][_x2](svfloat16x2_t zdn, svfloat16x2_t zm);
+  
+  
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x4_t svmaxnm[_f16][_x4](svfloat16x4_t zdn, svfloat16x4_t zm);
+  
+  
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x2_t svminnm[_f16][_x2](svfloat16x2_t zdn, svfloat16x2_t zm);
+  
+  
+  // Variants are also available for _f32 and _f64
+  __attribute__((arm_streaming))
+  svfloat16x4_t svminnm[_f16][_x4](svfloat16x4_t zdn, svfloat16x4_t zm);
+  
+  ```
+
+#### FRINTA, FRINTM, FRINTN, FRINTP
+
+Multi-vector floating-point round to integral value
+
+``` c
+  
+  __attribute__((arm_streaming))
+  svfloat32x2_t svrinta[_f32][_x2](svfloat32x2_t zn);
+  
+  
+  
+  __attribute__((arm_streaming))
+  svfloat32x4_t svrinta[_f32][_x4](svfloat32x4_t zn);
+  
+  
+  
+  __attribute__((arm_streaming))
+  svfloat32x2_t svrintm[_f32][_x2](svfloat32x2_t zn);
+  
+  
+  
+  __attribute__((arm_streaming))
+  svfloat32x4_t svrintm[_f32][_x4](svfloat32x4_t zn);
+  
+  
+  
+  __attribute__((arm_streaming))
+  svfloat32x2_t svrintn[_f32][_x2](svfloat32x2_t zn);
+  
+  
+  
+  __attribute__((arm_streaming))
+  svfloat32x4_t svrintn[_f32][_x4](svfloat32x4_t zn);
+  
+  
+  
+  __attribute__((arm_streaming))
+  svfloat32x2_t svrintp[_f32][_x2](svfloat32x2_t zn);
+  
+  
+  
+  __attribute__((arm_streaming))
+  svfloat32x4_t svrintp[_f32][_x4](svfloat32x4_t zn);
+  
+  ```
+
+#### LD1B, LD1D, LD1H, LD1W
+
+Contiguous load to multi-vector
+
+``` c
+  
+  __attribute__((arm_streaming))
+  svuint8x2_t svld1b_vnum[_u8]_x2(svcount_t png, const uint8_t *rn,
```

Comment:
Also, the convention for SVE has been to drop the access size from the instruction name unless the load is performing an extension (in which case the suffix is `sb` for signed extension from bytes and `ub` for unsigned extension from bytes).